packages/cli/src/lib/sandbox/docker.ts

```diff
@@ -180,13 +189,24 @@ export class DockerSandbox extends Sandbox {
       '-v',
       `${inputsPath}:/inputs.json:Z,ro`,
       '-v',
-      `${iterationJsonPath}:/task/description.json:Z,ro`,
+      `${iterationJsonPath}:/task/description.json:Z,ro`
+    );
+
+    // Mount initScript if provided in project config
+    if (projectConfig?.initScript) {
+      const initScriptAbsPath = join(projectRoot, projectConfig.initScript);
+      if (existsSync(initScriptAbsPath)) {
+        dockerArgs.push('-v', `${initScriptAbsPath}:/init-script.sh:Z,ro`);
+      }
```

Comment:
I would add a warning here if the init script does not exist (and if `--json` is false)

Reply:
Updated, took the opportunity to refactor how we manage the isJSON output in general, now we have the `global-state.ts` file that handles things like this (usually set once at the beginning of the program and unchanged for the rest of the execution)